components/select/Select.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import * as Styled from "./Select.styled";
+
+interface SelectProps {
+  children: React.ReactNode;
+  handleFilter: (filter: string) => void;
+}
+
+export const Select = ({ children, handleFilter }: SelectProps) => {
+  const handleChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    handleFilter(event.target.value);
+  };
+  return (
+    <Styled.SelectWrapper>
+      <Styled.Select onChange={handleChange}>
+        <option value="">Show all</option>
```

Comment:
I'd say some Selects won't have a default option that is called Show All, or have a default that have other functionality / values. 

Maybe best to also exclude this option from the component.